cve_bin_tool/util.py

```diff
@@ -7,37 +7,41 @@
 import sys
 from collections import defaultdict
 from enum import Enum
-from typing import NamedTuple
+from typing import List, NamedTuple, Optional, Set, Tuple, Union
 
 
 class OrderedEnum(Enum):
-    def __ge__(self, other):
+    def __ge__(self, other) -> bool:
```

Comment:
You've added return type but `other` is still untyped. It is the same type as the enclosing class so we will need [PEP 563](https://www.python.org/dev/peps/pep-0563/) for this to work.
Add this import at the top:
```py
from __future__ import annotations
```
and then you should be able to type all comparators like this:
```py
def __ge__(self, other: OrderedEnum) -> bool:
```